.github/workflows/CI.yml

```diff
@@ -95,32 +124,3 @@ jobs:
           .\.sonar\scanner\dotnet-sonarscanner begin /k:"samsmithnz_AzurePipelinesToGitHubActionsConverter" /o:"samsmithnz-github" /d:sonar.login="${{ secrets.SONAR_TOKEN }}" /d:sonar.host.url="https://sonarcloud.io"
           dotnet build src/AzurePipelinesToGitHubActionsConverter.Core/AzurePipelinesToGitHubActionsConverter.Core.csproj
           .\.sonar\scanner\dotnet-sonarscanner end /d:sonar.login="${{ secrets.SONAR_TOKEN }}"
-
-
-  NuGetPush:
```

Comment:
On one hand, you've optimized the build slightly, to not download artifacts, etc. On the other hand, now the "build" step is doing everything. Performance isn't that big of a deal either, the nuget push step runs in ~10s.

I'm torn here. Nothing technically wrong, more DevOps philosophy I think...

Reply:
Absolutely there is a time and place for multiple jobs for sure, I just don't think this is one.

Yes agreed it's more a philosophy thing;
- just because something is written as 'best practise' doesn't mean it's common sense or even plain practical.
- it reduces the number of lines of ci YAML significantly, which reduces surface area for possibility of bugs and reduces complexity
- at times of heavy loading your build job may complete, but due to the load you may have to wait XX seconds/minutes for a second agent to finally become available...